truffle-config.js

```diff
@@ -1,5 +1,15 @@
 const HDWalletProvider = require("@truffle/hdwallet-provider");
+const Web3 = require('web3')
+const ContractKit = require('@celo/contractkit')
+const web3 = new Web3('https://alfajores-forno.celo-testnet.org')
+const kit = ContractKit.newKitFromWeb3(web3)
+const getAccount = require('./getAccount').getAccount
 
+async function awaitWrapper(){
+    let account = await getAccount()
+    kit.connection.addAccount(account.privateKey)
+}
+awaitWrapper()
```

Comment:
just confirming: I believe this won't break another network's deployment / truffle configs, in case the dev didn't setup Celo's configs, correct?